rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cluster.put_settings/10_basic.yml

```diff
@@ -3,10 +3,10 @@
   - skip:
       version: " - 7.99.99"
       reason:  "transient settings deprecation"
-      features: "warnings"
+      features: "allowed_warnings"
```

Comment:
I think rather then using `allowed_warnings` it would be best to simply remove the warning in this test.

Reply:
Agreed, I should have explained when I made the PR. I'm keeping it here for now so my BWC tests work (rather than turning them off) since we have the change downstream. I'll follow-up with a PR to remove them completely once the change is backported all the way to 7.16.